00-RELEASENOTES

```diff
@@ -9,6 +9,27 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 8.1.2  -  Released Wed 04 June 2025
+================================================================================
+
+Upgrade urgency SECURITY: This release includes security fixes we recommend you
+apply as soon as possible.
```

Comment:
We should caveat this, since the CVE is only in the check-aof component.

Reply:
Agreed. Would HIGH make sense as it impacts subset of users?